scripts/js/upgrade/index.js

```diff
@@ -0,0 +1,330 @@
+    nonce = Number((await api.query.system.account(alice.address)).nonce);
+    const result = await councilProposeDemocracy(api, alice, preimageNoted, nonce)
+
+    console.log("Continuing with council vote using", result[0], result[1])
+    nonce = Number((await api.query.system.account(alice.address)).nonce);
+    await councilVoteProposal(api, alice, result[0], result[1], nonce, false)
+    nonce = Number((await api.query.system.account(bob.address)).nonce);
+    await councilVoteProposal(api, bob, result[0], result[1], nonce, false)
+    nonce = Number((await api.query.system.account(charlie.address)).nonce);
+    await councilVoteProposal(api, charlie, result[0], result[1], nonce, true)
+
+    console.log("Continuing to close council vote")
+    nonce = Number((await api.query.system.account(alice.address)).nonce);
+    const democracyIndex = await councilCloseProposal(api, alice, result[0], result[1], nonce)
+
+    console.log("Continuing with democracy vote on ref index", democracyIndex)
+    nonce = Number((await api.query.system.account(alice.address)).nonce);
+    await voteReferenda(api, alice, democracyIndex, nonce)
+
+    console.log("Waiting for referenda to be over and UpgradeAuthorized event is triggered")
+    await waitUntilEventFound(api, "UpgradeAuthorized")
+
+    console.log("Proceeding to enact upgrade")
+    nonce = Number((await api.query.system.account(alice.address)).nonce);
+    await enactUpgrade(api, alice, `0x${wasm.toString('hex')}`, nonce);
+
+    console.log("Waiting for ValidationFunctionApplied event")
+    await waitUntilEventFound(api, "ValidationFunctionApplied")
+
+    console.log("Waiting for 3 NewSession events")
+    let foundInBlock = await waitUntilEventFound(api, "EmptyTerm")
+    // await waitUntilEventFound(api, "NewSession")
+
+    console.log("getting header")
+    console.log("First event found, waiting for second event")
+    foundInBlock = await waitUntilEventFound(api, "EmptyTerm", foundInBlock+1)
+    // await waitUntilEventFound(api, "NewSession")
+
+    console.log("Second event found, waiting for third event")
+    await waitUntilEventFound(api, "EmptyTerm", foundInBlock+1)
+    // await waitUntilEventFound(api, "NewSession")
+
+    console.log("Runtime Upgrade succeeded")
+
+  } catch (error) {
+    console.log('error:', error);
+    exitCode = 1;
+  } finally {
+    await execCommand('cd ../../../ && ./scripts/init.sh stop-parachain-docker')
+    await execCommand('cd ../../../ && ./scripts/init.sh stop-relay-chain')
+    process.exit(exitCode)
+  }
+
+};
+
+async function execCommand(strCommand) {
+  try {
+    const { stdout, stderr } = await exec(strCommand);
+    console.log('stdout:', stdout);
+    console.log('stderr:', stderr);
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+async function waitUntilEventFound(api, eventName, fromBlock = 0) {
+  return new Promise(async (resolve, reject) => {
+    let maxCountDown = 30;
```

Comment:
Is this in seconds? if so, can you rename to
```suggestion
    let maxCountDownSecs = 30;
```